Makefile

```diff
@@ -228,11 +232,19 @@ install: whoami
 		--set ciip.namespace="$(CIIP_NAMESPACE_PREFIX)-$(ENVIRONMENT)" \
 		--set ciip.prefix="$(CIIP_NAMESPACE_PREFIX)" \
 		--set ciip.environment="$(ENVIRONMENT)" \
+		--set cif.release=cas-cif \
+		--set cif.namespace="$(CIF_NAMESPACE_PREFIX)-$(ENVIRONMENT)" \
+		--set cif.prefix="$(CIF_NAMESPACE_PREFIX)" \
+		--set cif.environment="$(ENVIRONMENT)" \
```

Comment:
This could use a future refactor. `namespace` can be derived from `prefix` and `environment`, and the `environment` should be the same across all applications